exchanges/bitfinex/bitfinex.go

```diff
@@ -655,93 +636,93 @@ func (b *Bitfinex) GetTickerBatch(ctx context.Context) (map[string]Ticker, error
 	for x := range response {
 		symbol, ok := response[x][0].(string)
 		if !ok {
-			return nil, common.GetTypeAssertError("string", response[x][0], "symbol")
+			return nil, common.GetTypeAssertError("string", response[x][0], "Ticker.Symbol")
 		}
 
 		var t Ticker
 		if len(response[x]) > 11 {
 			if t.FlashReturnRate, ok = response[x][1].(float64); !ok {
-				return nil, common.GetTypeAssertError("float64", response[x][1], "FlashReturnRate")
+				return nil, common.GetTypeAssertError("float64", response[x][1], "Ticker.Symbol.FlashReturnRate")
```

Comment:
Same here, these aren't part of a `Symbol` struct

Reply:
done